MeetupPlatformApi/Features/Authentication/RevokeUserRefreshTokens/RevokeUserRefreshTokensFeature.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace MeetupPlatformApi.Features.Authentication.RevokeUserRefreshTokens;
+
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Authentication)]
+public class RevokeUserRefreshTokensFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+
+    public RevokeUserRefreshTokensFeature(ApplicationContext context)
+    {
+        this.context = context;
+    }
+
+    [HttpDelete("/api/users/me/refresh-tokens")]
+    [Authorize]
+    public async Task<IActionResult> RevokeUserRefreshTokens()
+    {
+        var user = await context.Users.Include(user => user.RefreshTokens).SingleOrDefaultAsync(user => user.Id == CurrentUser.UserId);
```

Comment:
I'm not completely sure if it's needed (since I'm using a web view right now), but we might want to put method calls on a new line, like so:
```csharp
var user = await context.Users
    .Include(user => user.RefreshTokens)
    .SingleOrDefaultAsync(user => user.Id == CurrentUser.UserId);
```
But it's up to You guys to decide